src/VisualStudio/CSharp/Impl/Options/CSharpSettingsManagerOptionSerializer.cs

```diff
@@ -65,6 +65,7 @@ public CSharpSettingsManagerOptionSerializer(VisualStudioWorkspaceImpl workspace
         private const string Style_UseImplicitTypeWherePossible = nameof(AutomationObject.Style_UseImplicitTypeWherePossible);
         private const string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration = nameof(AutomationObject.Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration);
         private const string Style_PreferIntrinsicPredefinedTypeKeywordInMemberAccess = nameof(AutomationObject.Style_PreferIntrinsicPredefinedTypeKeywordInMemberAccess);
+        private const string Style_UseBracesWherePossible = nameof(AutomationObject.Style_UseBracesWherePossible);
 
```

Comment:
Could be changed.  Or in order to not have to reverse all my checking logic, it could be "Always use braces" or something.